src/functionaltests/java/com/ericsson/ei/utils/TestConfigs.java

```diff
@@ -1,62 +1,53 @@
 package com.ericsson.ei.utils;
 
 import com.mongodb.MongoClient;
-import com.rabbitmq.client.Connection;
-import com.rabbitmq.client.ConnectionFactory;
 
 import java.io.File;
 import java.io.IOException;
+import java.util.HashMap;
 
 import org.apache.tomcat.util.codec.binary.Base64;
 import org.apache.tomcat.util.codec.binary.StringUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.amqp.core.BindingBuilder;
-import org.springframework.amqp.core.Queue;
-import org.springframework.amqp.core.TopicExchange;
-import org.springframework.amqp.rabbit.connection.CachingConnectionFactory;
-import org.springframework.amqp.rabbit.core.RabbitAdmin;
 import org.springframework.util.SocketUtils;
 
 import de.flapdoodle.embed.mongo.distribution.Version;
 import de.flapdoodle.embed.mongo.tests.MongodForTestsFactory;
 
 public class TestConfigs {
 
-    private static AMQPBrokerManager amqpBroker;
-    private static ConnectionFactory cf;
-    private Connection conn;
-    private MongodForTestsFactory testsFactory;
+    final static Logger LOGGER = (Logger) LoggerFactory.getLogger(TestConfigs.class);
+
     private MongoClient mongoClient = null;
 
-    final static Logger LOGGER = (Logger) LoggerFactory.getLogger(TestConfigs.class);
+    protected static HashMap<Integer, AMQPBrokerManager> amqpBrokerPool = new HashMap<Integer, AMQPBrokerManager>();
```

Comment:
should be  - Map < Integer, AMQPBrokerManager > amqpBrokerPool = new HashMap<>();
we should use interface realization for collections and diamonds type in new HashMap<>